client/features/auth-pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import useDocumentTitle from '@hooks/useDocumentTitle';
+import { Link as RouterLink } from 'react-router-dom';
+import { Button, Box, Typography, Link, TextField } from '@mui/material';
+import styles from '@styles/layout/AuthStyles.module.css';
+import { zodResolver } from '@hookform/resolvers/zod';
+import type { SubmitHandler } from 'react-hook-form';
+import { useForm } from 'react-hook-form';
+import { useGetPasswordResetLinkMutation } from '@/hooks/useGetPasswordResetLinkMutation';
+import { userEmailSchema } from '@/zod/validateUseremail';
+import SnackBarNotification from '@components/common/SnackBarNotification';
+
+interface FormData {
+  email: string;
+}
+
+const ForgotPasswordPage = () => {
+  useDocumentTitle('Forgot Password');
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>({
+    resolver: zodResolver(userEmailSchema),
+  });
+
+  const { mutate, isPending, isSuccess } = useGetPasswordResetLinkMutation();
+
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState('');
+  const [snackbarSeverity, setSnackbarSeverity] = useState<
+    'error' | 'warning' | 'info' | 'success'
+  >('info');
+
+  const handleCloseSnackbar = () => {
```

Comment:
Maybe a good practice to also clear the snackbarMessage and snackbarSeverity state when closing the snack bar?